cli/src/test/java/com/devonfw/tools/ide/io/FileAccessImplTest.java

```diff
@@ -675,5 +675,102 @@ public void testFindExistingFileInFolders(@TempDir Path tempDir) {
   private void postExtract(Path path) {
   }
 
+  /**
+   * Test of {@link FileAccessImpl#symlink(Path, Path, boolean)} when broken junctions exist. This simulates the scenario
+   * described in issue #1169 where mklink fails with "Cannot create a file when that file already exists" when trying
+   * to create a junction over a broken junction.
+   */
+  @Test
+  public void testSymlinkOverwritesBrokenJunction(@TempDir Path tempDir) throws IOException {
+
+    // arrange
+    IdeContext context = IdeTestContextMock.get();
+    if (!windowsJunctionsAreUsed(context, tempDir)) {
```

Comment:
I was not even aware of this existing method `windowsJunctionsAreUsed`.
IMHO this is odd and here we should just use `@EnabledOnOs({OS.WINDOWS})` on the test method.
Am I missing something?

Reply:
externalized to #1422